udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -37,7 +37,7 @@
 {% endblock %}
 
 {% block content %}
-{% cache cache_duration, 'dataset-content', dataset.id|string, g.lang_code, current_user.slug or 'anonymous' %}
+{% cache cache_duration, 'dataset-content', dataset.id|string + reuses.page|string, g.lang_code, current_user.slug or 'anonymous' %}
```

Comment:
All the keys are getting concatenated I think, so we can give it as a list?
```suggestion
{% cache cache_duration, 'dataset-content', dataset.id|string, reuses.page|string, g.lang_code, current_user.slug or 'anonymous' %}
```

Reply:
Done !